app/assets/stylesheets/layout/_how-it-works.scss

```diff
@@ -0,0 +1,16 @@
+.how-it-works__inf {
+  text-align: center;
+}
+
+.how-it-works__category {
+  @media (min-width: 768px) {
+    display: flex;
+  };
+}
+
+.how-it-works__description {
+  img {
+    @include make-col(6);
+    padding: 30px 0;
+  }
+}
```

Comment:
I think it is a good practice to add an empty line when content of the file ends.